.cproject

```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I'm not too experienced in working with Git outside the norm.  When I checked them in, they got added to the pull-request.  You then asked for them to be removed from the distribution ... so I added them to gitignore.  However, this means that they are still in the pull request.  The only way I know to remove them would be for the pull-request to be completed, you to sync your clone, remove them in your clone and then you commit your project with the files removed.  My environment will no longer attempt to try and include them in future commits.